openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -198,25 +198,36 @@
   narrative:
     - key: a
       text: |
-        Organizational review and analysis of information system
-        audit records for indications of organization-defined
-        inappropriate or unusual activity is outside the scope
-        of OpenShift configuration.
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
     - key: b
       text: |
-        Reporting findings of organization-defined inappropriate or
-        unusual activity to organization-defined personnel or roles is
-        outside the scope of OpenShift configuration.
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
```

Comment:
The text of AU-6(b) from https://nvd.nist.gov/800-53/Rev4/control/Au-6 is:
`` Reports findings to [Assignment: organization-defined personnel or roles].``

How does this new language address reporting any findings of unusual behavior to appropriate contacts?